x-pack/solutions/security/plugins/security_solution/server/endpoint/services/actions/clients/microsoft/defender/endpoint/ms_defender_endpoint_actions_client.ts

```diff
@@ -705,7 +715,11 @@ export class MicrosoftDefenderEndpointActionsClient extends ResponseActionsClien
                   ? actionRequest.agent.id[0]
                   : actionRequest.agent.id,
                 data: { command: actionRequest.EndpointActions.data.command },
-                error: isError ? { message } : undefined,
+                error: isError
+                  ? {
+                      message: commandErrors.length > 0 ? commandErrors[0] : message,
```

Comment:
This looks strange to me. So you added code above to aggregate all of the errors, but here you are only including the first error in that array. Should this instead include all of the errors?

Reply:
Good point 👍  I'll return it directly from the `machineAction`

Reply:
I realized that these errors come in the 'opposite' order, so we actually render the last one. 

I think it's pretty safe to render the it as it is - get the list and render the first one from it. 
WDYT?

Reply:
I'm not sure I understand. I don't see any changes from the last comment I provided and you still seem to be loping through all errors to put them into an array only then show the first one in that array. Am I missing something?